api/.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
```

Comment:
Should we do the same Prettier trick for ESLint? I'd like the sub-packages (under `/packages`) and the app to adhere to the same linter rules 🙂

Reply:
Done!